pylint_odoo/checkers/modules_odoo.py

```diff
@@ -114,6 +128,15 @@
 DFLT_EXTFILES_CONVERT = ['csv', 'sql', 'xml', 'yml']
 DFLT_EXTFILES_TO_LINT = DFLT_EXTFILES_CONVERT + [
     'po', 'js', 'mako', 'rst', 'md', 'markdown']
+DFLT_IMPORT_NAME_WHITELIST = [
+    # self-odoo
+    'odoo', 'openerp',
+    # Known external packages of odoo
+    'PIL', 'babel', 'dateutil', 'decorator', 'docutils', 'jinja2', 'ldap',
```

Comment:
(Don't worry.)

Yes, that is the intention.
Here we have a "Deafault" value for initial assignation from configuration file.
But we can create a configuration file by each version with this value but is from running step [MQT configuration files of pylint](https://github.com/OCA/maintainer-quality-tools/tree/master/travis/cfg)